linera-views/src/backends/value_splitting.rs

```diff
@@ -255,52 +269,55 @@ where
                 }
             }
         }
-        self.store.write_batch(batch_new).await
+        Ok(self.store.write_batch(batch_new).await?)
     }
 
     async fn clear_journal(&self) -> Result<(), Self::Error> {
-        self.store.clear_journal().await
+        Ok(self.store.clear_journal().await?)
     }
 }
 
 impl<K> ValueSplittingStore<K>
 where
     K: RestrictedKeyValueStore + Send + Sync,
-    K::Error: From<bcs::Error> + From<DatabaseConsistencyError>,
+    K::Error: 'static,
 {
     /// Creates a new store that deals with big values from one that does not.
     pub fn new(store: K) -> Self {
         ValueSplittingStore { store }
     }
 
-    fn read_count_from_value(value: &[u8]) -> Result<u32, K::Error> {
+    fn read_count_from_value(value: &[u8]) -> Result<u32, <Self as WithError>::Error> {
```

Comment:
`ValueSplittingError<K::Error>` also possible instead of `<Self as WithError>::Error`

Reply:
Ok, done.